hoot-core/src/main/cpp/hoot/core/io/db/PostgresqlDumpfileWriter.cpp

```diff
@@ -452,15 +428,19 @@ std::list<QString> PostgresqlDumpfileWriter::_createSectionNameList()
 
 void PostgresqlDumpfileWriter::_createNodeTables()
 {
-  _createTable( ApiDb::getCurrentNodesTableName(),
-                "COPY " + ApiDb::getCurrentNodesTableName() + " (id, latitude, longitude, changeset_id, visible, \"timestamp\", tile, version) FROM stdin;\n" );
-  _createTable( ApiDb::getCurrentNodeTagsTableName(),
-                "COPY " + ApiDb::getCurrentNodeTagsTableName() + " (node_id, k, v) FROM stdin;\n" );
+  _createTable(ApiDb::getCurrentNodesTableName(),
+                "COPY " + ApiDb::getCurrentNodesTableName() +
+               " (id, latitude, longitude, changeset_id, visible, \"timestamp\", tile, version) FROM stdin;\n" );
+  _createTable(ApiDb::getCurrentNodeTagsTableName(),
+                "COPY " + ApiDb::getCurrentNodeTagsTableName() +
+               " (node_id, k, v) FROM stdin;\n" );
 
-  _createTable( ApiDb::getNodesTableName(),
-                "COPY " + ApiDb::getNodesTableName() + " (node_id, latitude, longitude, changeset_id, visible, \"timestamp\", tile, version, redaction_id) FROM stdin;\n" );
-  _createTable( ApiDb::getNodeTagsTableName(),
-                "COPY " + ApiDb::getNodeTagsTableName() + " (node_id, version, k, v) FROM stdin;\n" );
+  _createTable(ApiDb::getNodesTableName(),
+                "COPY " + ApiDb::getNodesTableName() +
+               " (node_id, latitude, longitude, changeset_id, visible, \"timestamp\", tile, version, redaction_id) FROM stdin;\n" );
+  _createTable(ApiDb::getNodeTagsTableName(),
+                "COPY " + ApiDb::getNodeTagsTableName() +
+               " (node_id, version, k, v) FROM stdin;\n" );
```

Comment:
Yeah, sorry.  I'm still cleaning up Terry's lines running over 100 spaces and the spaces between parans.  I guess we need some type of style checker like the java/js have, so that stuff doesn't get in there in the first place.